packages/config/code-hike.scss

```diff
@@ -70,6 +70,9 @@
   //   font-family: Ubuntu, Droid Sans, -apple-system, BlinkMacSystemFont, Segoe WPC, Segoe UI,
   // sans-serif;
 }
+.ch-code-wrapper {
+  background-color: var(--ch-t-background) !important;
```

Comment:
@saltcod Not sure where `--ch-t-background` comes from.
I'm seeing the code snippets basically without a bg in the blog posts.

<img width="795" alt="Screenshot 2023-10-26 at 16 06 17" src="https://github.com/supabase/supabase/assets/25671831/32fcf806-1e32-44db-b600-e85a8bfba6e8">

Can I use the previous `--ch-16` here?

Reply:
With `--ch-16`:

<img width="767" alt="Screenshot 2023-10-26 at 16 09 41" src="https://github.com/supabase/supabase/assets/25671831/099a065a-5a8d-4892-81cb-28e18a90d225">